card/mod/03_machines/lib/javascript/wagn_mod.js.coffee

```diff
@@ -37,7 +37,7 @@ $.extend wagn,
     wagn.tinyMCEConfig = setter()
   
   initAce: (textarea) ->
-    type_code = textarea.slot().attr "data-card-type-code"
+    type_code = textarea.attr "data_card_type_code"
```

Comment:
if I understand correctly, it's actually kind of important that it be "data-" rather than "data_".  See http://www.w3schools.com/tags/att_global_data.asp

Either data-type_code or data-type-code is fine.